src/main/java/no/nav/tag/kontaktskjema/gsak/GsakScheduler.java

```diff
@@ -0,0 +1,22 @@
+package no.nav.tag.kontaktskjema.gsak;
+
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import lombok.extern.slf4j.Slf4j;
+import net.javacrumbs.shedlock.core.SchedulerLock;
+
+@Slf4j
+@Component
+public class GsakScheduler {
+
+    private static final String ONE_MIN = "PT14M";
+
+    private static final String THIRTY_SECONDS = "PT30S";
+
+    @Scheduled(cron = "* * * * * ?")
+    @SchedulerLock(name = "navn", lockAtMostForString = ONE_MIN, lockAtLeastForString = THIRTY_SECONDS)
+    public void scheduledBehandleSkjemaer() {
```

Comment:
Denne jobben kjører nå hvert sekund, men låses i et halvt minutt. Dermed vil den i praksis kjøres en gang hvert halve minutt med dette oppsettet. Vi trenger neppe å kjøre så ofte for å opprette oppgaver i gsak.